js/app.js

```diff
@@ -117,17 +144,54 @@ $("#mc-embedded-subscribe-form").ajaxForm({
 
       return;
     }
-    var amount = $('input[name="AMOUNT"]:checked').val();
-    $('#amount-to-donate').text('$' + amount);
-    $('#mc-embedded-subscribe-form').addClass('hide');
-    $('#confirmation').removeClass('hide').hide().fadeIn();
-    window.scrollTo(0, $('#confirmation').offset().top - 30);
+  }
 
-    ga('send', 'event', {
-      eventCategory: 'subscription',
-      eventAction: 'success',
-      eventLabel: 'join flow',
-      nonInteraction: true,
-    });
+  var amount = $('input[name="AMOUNT"]:checked').val();
+  $('#mc-submit-button').text("Loading...");
+
+  ga('send', 'event', {
+    eventCategory: 'subscription',
+    eventAction: 'success',
+    eventLabel: 'join flow',
+    nonInteraction: true,
+  });
+
+  paypalAction(amount);
+}
+
+// Pre-fill form values
+function getUrlParameter(sParam) {
+  var sPageURL = decodeURIComponent(window.location.search.substring(1)),
+    sURLVariables = sPageURL.split('&'),
+    sParameterName,
+    i;
+
+  for (i = 0; i < sURLVariables.length; i++) {
+    sParameterName = sURLVariables[i].split('=');
+
+    if (sParameterName[0] === sParam) {
+      return sParameterName[1] === undefined ? true : sParameterName[1];
```

Comment:
I'm not 100% sure, but does this mean if I set no value, e.g. `amount=&name=&` this will return `true` for amount and name?

And in that case, does that mean that a set of empty params will result in the alert showing? Is that desired behavior?